core/src/com/unciv/ui/worldscreen/TechPolicyDiplomacyButtons.kt

```diff
@@ -56,12 +60,20 @@ class TechPolicyDiplomacyButtons(val worldScreen: WorldScreen) : Table(BaseScree
         diplomacyButtonHolder.onClick {
             game.pushScreen(DiplomacyScreen(viewingCiv))
         }
+        if (game.gameInfo!!.isEspionageEnabled()) {
+            espionageButton.add(ImageGetter.getImage("OtherIcons/Spy_White")).size(30f).pad(15f)
+            espionageButtonHolder.onClick {
+                game.pushScreen(EspionageOverviewScreen(viewingCiv))
+            }
+        }
```

Comment:
I have enabled the Espionage in the Debug menu, started a new game with enabled the Espionage option, `game.gameInfo!!.isEspionageEnabled()` is True here, but I still do not see the button.
Where should it be?

Reply:
It shows up once you have a spy, which happens when anyone enters the Renaissance era, should be next to the diplomacy and policy buttons

Reply:
Never mind, I've found out that it appears when you have at least 1 spy, and to get it you should enter the Renessance Era by a natural way. I was creating a game in the Informational Era from the very beginning.